client/src/app/site/mediafiles/components/mediafile-list/mediafile-list.component.ts

```diff
@@ -216,18 +218,24 @@ export class MediafileListComponent extends BaseListViewComponent<ViewMediafile>
      * Set the title, make the edit Form and observe Mediafiles
      */
     public ngOnInit(): void {
+        super.ngOnInit();
         super.setTitle('Files');
 
         this.repo.getDirectoryIdByPath(this.route.snapshot.url.map(x => x.path)).then(directoryId => {
             this.changeDirectory(directoryId);
         });
 
         // Observe the logo actions
+        /**
```

Comment:
The logo/font handling has changed completly: It should also be done in this PR.

Mediafiles have template fields `used_as_logo_$_in_meeting` and `used_as_font_$_in_meeting`. The values to replace are the places for the mediafiles. E.g. for logos there can be `web_header`, `pdf_header`, `pdf_footer`, ... (see https://github.com/OpenSlides/OpenSlides/blob/master/openslides/core/config_variables.py#L255 without the `logo_` prefix)

This list must now be defined in the client - the server does not know about these fields anymore. It implies, that the client can define fields, as it likes without server changes.

So, if a mediafile is used as the `web_header` and `pdf_header`, the fields `used_as_logo_web_header_in_meeting` and `used_as_logo_pdf_header_in_meeting` are set.

Maybe the `MediaManageService` can save the list of possible places and do some handling of that.

Reply:
For selecting a place in the dot-menu: There will be a dedicated action for setting a mediafile in a meeting as X (e.g. `projector_header`).

Reply:
It's still not clear how and from where to get the information about the currently used logo(s) and fonts, this this feature is way beyond the scope of this pull request.